src/transport/RendezvousSession.cpp

```diff
@@ -181,7 +181,24 @@ void RendezvousSession::OnPairingComplete()
         return;
     }
 
-    UpdateState(State::kNetworkProvisioning);
+    if (mParams.GetPeerAddress().GetTransportType() != Transport::Type::kBle || // For rendezvous initializer
```

Comment:
maybe a helper in PeerAddress to say 'IsIpType()' may make this simpler. I would prefer testing for 'is ip' instead of 'is not ble' to make the intent clear.

Reply:
The check for BLE address is a temporary solution, we will remove the check for BLE transport in the future after network provisioning is ready.